openapi/openapi.json

```diff
@@ -27072,6 +27072,12 @@
         "forceRedeploymentReason"
       ],
       "properties": {
+        "backendQuotaGiB": {
+          "description": "backendQuotaGiB sets the etcd backend storage size limit in gibibytes. The value should be an integer not less than 8 and not more than 32. When not specified, the default value is 8.",
+          "type": "integer",
+          "format": "int32",
+          "default": 0
```

Comment:
That's not right, this should be 8, whys the tag not worked 🤔

Reply:
I have no idea why .. the taga are correct